packages/components/src/Text/Text.tsx

```diff
@@ -30,7 +32,11 @@ export function Text({
   };
 
   return (
-    <Typography textColor={textColors[variation] as TextColor} size={size}>
+    <Typography
+      textColor={textColors[variation] as TextColor}
+      size={size}
+      numberOfLines={numberOfLines || undefined}
```

Comment:
do we need to explicitly set this fallback here?

if we do need it, an alternative approach might be to set this as a default of `undefined` up on line 23